webapp/src/widgets/label.tsx

```diff
@@ -13,9 +15,13 @@ type Props = {
 
 // Switch is an on-off style switch / checkbox
 function Label(props: Props): JSX.Element {
+    let color = 'empty'
+    if (Constants.menuColors.hasOwnProperty(props.color || '')) {
+        color = props.color || 'empty'
```

Comment:
Already init `color = 'empty'` so I don't think the `||` is necessary here.

Reply:
again is about `string|undefined` type.